internal/pkg/deploy/cloudformation/stack/backend_svc.go

```diff
@@ -81,10 +81,19 @@ func (s *BackendService) Template() (string, error) {
 	if err != nil {
 		return "", err
 	}
-	sidecars, err := convertSidecar(s.manifest.Sidecars)
+	convSidecarOpts := convertSidecarOpts{
+		sidecarConfig: s.manifest.Sidecars,
+		imageConfig:   &s.manifest.ImageConfig.Image,
+		workloadName:  *s.manifest.Name,
```

Comment:
nit: prefer `aws.*Value` functions to naked pointer dereference
```suggestion
		workloadName:  aws.StringValue(s.manifest.Name),
```